yourgov/components/Staff/StaffMembersTable.tsx

```diff
@@ -97,6 +97,14 @@ export const StaffMembersTable = ({
 		return updatedStaffMember?.name || '';
 	});
 
+	useEffect(() => {
+		if (successMessageType && updatedStaffMemberName) {
+			setTimeout(() => {
```

Comment:
Is it worth clearing this timeout on unmount?

Reply:
Maybe?